### ISSUE TITLE
Consider how to convey the facet values in facets config dashboard

### ISSUE THREAD

**Issue by jkeck** [projectblacklight/spotlight#2449]
This icon that currently displays all facet values will be removed as part of the a11y work being undertaken for 3.0.0 (the icon is inaccessible and not the best UI for this kind of information).

See sul-dlss/exhibits#1630
